tests/integration/kubernetes/confidential/confidential_common.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "../../tests_common.bash"
+
+SUPPORTED_HYPERVISORS="qemu-sev qemu-snp"
+
+setup_unencrypted_confidential_pod() {
+    local pod_name="unencrypted-confidential"
+    local IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+    local UNENCRYPTED_IMAGE_URL="${IMAGE_REPO}:unencrypted"
+    local TEST_DIR="$(mktemp -d /tmp/test-kata-confidential.XXXXXXXX)"
+    export SSH_KEY_FILE="${TEST_DIR}/container-ssh-key"
+
+    if [[ "$SUPPORTED_HYPERVISORS" == *"${KATA_HYPERVISOR}"* ]]; then
+        # Pull image
+        docker pull "${UNENCRYPTED_IMAGE_URL}"
+        
+        # Get ssh key from docker image label and save to file
+        docker inspect ${image_url} \
+            | jq -r '.[0].Config.Labels.ssh_key' \
+            | sed "s|\(-----BEGIN OPENSSH PRIVATE KEY-----\)|\1\n|g" \
+            | sed "s|\(-----END OPENSSH PRIVATE KEY-----\)|\n\1|g" \
+            > "${SSH_KEY_FILE}"
+
+        # Set permissions on private key file
+        chmod 600 "${SSH_KEY_FILE}"
+
+        if [[ "${KATA_HYPERVISOR}" == "qemu-sev" ]]; then
+            export REMOTE_COMMAND="sudo dmesg | grep SEV"
+        elif [[ "${KATA_HYPERVISOR}" == "qemu-snp" ]]; then
+            export REMOTE_COMMAND="sudo dmesg | grep SNP"
+        fi
```

Comment:
... as here we can simply then do something like:
```sh
export REMOTE_COMMAND="${REMOTE_COMMAND_PER_HYPERVISOR[${KATA_HYPERVISOR}]"
```

Reply:
Changed.